bitnami/magento/2/debian-12/rootfs/opt/bitnami/scripts/libmagento.sh

```diff
@@ -228,6 +265,10 @@ magento_initialize() {
             info "Trying to connect to Elasticsearch"
             magento_wait_for_es_connection "$es_host" "$es_port"
         fi
+        if [[ "$MAGENTO_SEARCH_ENGINE" =~ ^opensearch ]]; then
+            info "Trying to connect to Opensearch"
+            magento_wait_for_es_connection "$oss_host" "$oss_port"
```

Comment:
I would rename this function to something generic. E.g. `magento_wait_for_search_engine_connection` or similar.